src/uu/rm/src/rm.rs

```diff
@@ -128,6 +128,10 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
             .long(OPT_RECURSIVE)
             .help("remove directories and their contents recursively")
         )
+        .arg(
```

Comment:
I think you could use the --alias option ?
https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.alias

Reply:
I also thought clap's `visible_alias` function could be use for this. However, I as far as I can tell it is not possible to mimic GNU's behaviour with `alias`. The problem is that alias is only for the long flag, i.e. with `.visible_alias("R")` the user would have to type `--R`.

Reply:
OK, TIL :)

Could you please add a comment to make it explicit in the code? thanks

Reply:
done